core/src/utils/FilesystemUtils.hpp

```diff
@@ -28,20 +28,22 @@
  * SOFTWARE.
  *
  */
+#ifndef LEDGER_CORE_FILESYSTEMUTILS_HPP
+#define LEDGER_CORE_FILESYSTEMUTILS_HPP
```

Comment:
We use `#pragma once` as much as possible now, so when we change the include guards we change to pragma

Reply:
fixed